File: Latihan b/cpp/hardware.cpp
// Include
#include <iostream>
#include <string>
#include "product.cpp"
using namespace std;
// deklarasi kelas hardware inherit produk
class hardware : public product{
    // variabel privat
    private:
        string brand;
        string model;
    // method public dan konstruktor
    public:
        hardware(){

        }
        // get dans set semua variabel
        void setbrand(string brand){
            this->brand = brand;
        }
        string getbrand(){
            return this->brand;
        }
        void setmodel(string model){
            this->model = model;
        }
        string getmodel(){
            return this->model;
        }
        // desktruktor
        ~hardware(){

        }
};

File: Latihan b/Python/hardware.py
# Include/import
from product import product
# deklarasi kelas hardware inherit produk
class hardware(product):
    # method dan konstruktor
    def __init__(self):
        self.brand = ""
        self.model = ""
    #  get dans set semua variabel
    def setbrand(self, brand):
        self.brand = brand
    def getbrand(self):
        return self.brand
    def setmodel(self, model):
        self.model = model
    def getmodel(self):
        return self.model